tests/poutine/test_mapdata.py

```diff
@@ -293,39 +285,45 @@ def irange_cuda_model(subsample_size):
 def map_data_vector_cuda_model(subsample_size):
     mu = Variable(torch.zeros(20).cuda())
     sigma = Variable(torch.ones(20).cuda())
-    pyro.map_data("data", mu,
-                  lambda i, mu: pyro.sample("x", dist.normal, mu, sigma[i]),
-                  batch_size=subsample_size, use_cuda=True)
+    pyro.map_data(
+        "data", mu, lambda i, mu: pyro.sample("x", dist.normal, mu, sigma[i]), batch_size=subsample_size, use_cuda=True)
 
 
 def map_data_iter_cuda_model(subsample_size):
     mu = Variable(torch.zeros(20).cuda())
     sigma = Variable(torch.ones(20).cuda())
-    pyro.map_data("data", list(mu),
-                  lambda i, mu: pyro.sample("x_{}".format(i), dist.normal, mu, sigma[i]),
-                  batch_size=subsample_size, use_cuda=True)
+    pyro.map_data(
+        "data",
+        list(mu),
+        lambda i, mu: pyro.sample("x_{}".format(i), dist.normal, mu, sigma[i]),
+        batch_size=subsample_size,
+        use_cuda=True)
 
 
 @requires_cuda
 @pytest.mark.parametrize('subsample_size', [5, 20])
-@pytest.mark.parametrize('model', [
-    iarange_cuda_model,
-    irange_cuda_model,
-    map_data_vector_cuda_model,
-    map_data_iter_cuda_model,
-], ids=["iarange", "irange", "map_data_vector", "map_data_iter"])
+@pytest.mark.parametrize(
+    'model', [
+        iarange_cuda_model,
+        irange_cuda_model,
+        map_data_vector_cuda_model,
+        map_data_iter_cuda_model,
+    ],
+    ids=["iarange", "irange", "map_data_vector", "map_data_iter"])
 def test_cuda(model, subsample_size):
     tr = poutine.trace(model).get_trace(subsample_size)
     assert tr.log_pdf().is_cuda
     assert tr.batch_log_pdf().is_cuda
 
 
-@pytest.mark.parametrize('model', [
-    iarange_model,
-    irange_model,
-    map_data_vector_model,
-    map_data_iter_model,
-], ids=['iarange', 'irange', 'map_data_vector', 'map_data_iter'])
+@pytest.mark.parametrize(
```

Comment:
Please either revert the autoformat changes or split them into a separate PR.

Reply:
My bad, this should have been two different commits. Will revert and push again.